pkg/proc/test/support.go

```diff
@@ -81,6 +81,12 @@ const (
 
 // BuildFixture will compile the fixture 'name' using the provided build flags.
 func BuildFixture(name string, flags BuildFlags) Fixture {
+	return BuildFixtureWithFlags(name, flags, "")
+}
+
+// BuildFixtureWithFlags will compile the fixture 'name' using the provided predefined
+// and custom build flags.
+func BuildFixtureWithFlags(name string, flags BuildFlags, andflags string) Fixture {
```

Comment:
Maybe call it customFlags to match the comment? Also, the comment should say what the format for the flags is. "Flags" is plural but the type is `string`, so it should be more specific (space separated, etc.)

Reply:
I ended with a different rename because I rewrote the comment. Please take a look.